### ISSUE TITLE
This has been a great library - it appears now unmaintained

### ISSUE THREAD

**Issue by waynebrantley** [jquense/react-widgets#1158]
Looks like react 19 does not work with it and with 1 commit in the last 33 months I think it may be time to find another library.
Any thoughts/ideas from the community?

**Comment by jquense** [comment]
it's not unmaintained, it's just "done" as in it works and I'm pretty happy with the state of it. I am still working my way through my stack of OSS to upgrade things to react 19, but it'll happen. What specifically doesn't work?

**Comment by waynebrantley** [comment]
I saw this issue:  https://github.com/jquense/react-widgets/issues/1157
With only 1 commit in more a little over a year and the commit prior to that was from almost 3 years ago, it appeared to be unmaintained - @jquense glad you are still maintaining it,    It has been a good library!

**Comment by jquense** [comment]
should be fixed in 5.8.5, lmk if something doesn't work

**Comment by ghost1face** [comment]
I just pulled the latest and it appears the `scss` folder is incorrectly structured:

![image](https://github.com/user-attachments/assets/1647103a-2162-4a23-a6b4-936aa49de50d)

It looks as though the folder structure was changed from `scss/styles.scss` to `scss/src/scss/styles.scss`.  If this was intended I can adjust but want to call it out just in case this was not the intention.

Thanks for the React19 compatibility updates!
